LuaSkin/LuaSkin/Skin.h

```diff
@@ -43,10 +47,14 @@
 #define LS_TFUNCTION      1 << 7
 /*! @define LS_TUSERDATA maps to LUA_TUSERDATA */
 #define LS_TUSERDATA      1 << 8
-/*! @define LS_TNONE maps to LUA_TNONE */
+/*! @define LS_TNONE maps to LUA_TNONE.  Deprecated, as this serves no real use in checkArgs except to provide parity with Lua's LUA_TNONE, which is handled by optional argument tagging or as an argument count error. */
 #define LS_TNONE          1 << 9
 /*! @define LS_TANY indicates that any Lua variable type is accepted */
 #define LS_TANY           1 << 10
+/*! @define LS_TINTEGER Can be OR'd with LS_TNUMBER to specify an integer argument.  This option is ignored if paired with other types. */
+#define LS_TINTEGER       1 << 11
+/*! @define LS_TSOFTBREAK Similar to LS_TBREAK, in that it signals the end of the argument list @link //apple_ref/occ/instm/LuaSkin/checkArgs: checkArgs @/link should validate; however, the argument count is not verified, so any additional arguments beyond the validated list are the responsibility of the module function to check and use or ignore. */
```

Comment:
Thanks. It should probably also be explicitly documented that this only works as the final argument (and afaict from the code, it shouldn't matter if `LS_TBREAK` is specified afterwards, so we probably don't need to explicitly say anything about whether or not `LS_TBREAK` should follow it)